samples/projects/awscdk-typescript/lib/config.json

```diff
@@ -20,36 +20,26 @@
       }
     ]
   },
-  "components": [
-    {
-      "name": "static-website-hosting",
-      "type": "distribution",
-      "subtype": "staticWebsite",
-      "details": [
-        {
-          "name": "test-static-website",
-          "hostName": "staticsite",
-          "domain": "test.fabrhq.dev"
-        }
-      ]
-    }
-  ],
   "services": [
     {
-      "name": "ecs-farget",
+      "name": "ecs-fargate",
       "type": "container",
       "subtype": "ecs",
       "details": [
         {
           "name": "web-service",
           "application": "PUBLIC",
-          "dockerfilePath": "/test-apps/api",
+          "containerRegistry": "dockerhub",
+          "containerImageName": "janaka/test-api",
```

Comment:
I wonder if we could take use of GH Packages to avoid the extra dependency on Dockerhub?

Reply:
We can, I used DockerHub for this initial cut because it's the simplest I know. I want to add support for other registries. 

TIL GH Packages had container support

Reply:
Let's do it with next PR